File: app/src/main/java/org/schulcloud/mobile/data/model/Comment.java
package org.schulcloud.mobile.data.model;

import io.realm.RealmModel;
import io.realm.annotations.PrimaryKey;
import io.realm.annotations.RealmClass;

@RealmClass
public class Comment implements RealmModel {
    @PrimaryKey
    public String _id;
    public String comment;
    public String submissionId;
    public String author;
    public String createdAt;
}

File: app/src/main/java/org/schulcloud/mobile/models/homework/submission/Comment.kt
package org.schulcloud.mobile.models.homework.submission

import com.google.gson.annotations.SerializedName
import io.realm.RealmObject
import io.realm.annotations.PrimaryKey
import org.schulcloud.mobile.models.base.HasId

open class Comment : RealmObject(), HasId {

    @PrimaryKey
    @SerializedName("_id")
    override var id: String = ""

    var submissionId: String? = null
    var author: String? = null
    var comment: String? = null
    var createdAt: String? = null

}
